cla_public/apps/checker/session.py

```diff
@@ -182,6 +182,22 @@ def api_payload(self):
 
         return Notes()
 
+    @property
+    def callback_requested(self):
+        return self.is_yes('CallMeBackForm', 'callback_requested')
+
+    def clear_and_store_ref(self):
+        if not self.get('is_expired', False):
+            store = {
+                'is_expired': True,
+                'stored_case_ref': self.get('case_ref'),
+                'stored_callback_time': self.callback_time,
+                'stored_callback_requested': self.callback_requested,
+                'stored_category': self.category,
+            }
+            self.clear()
+            self.update(store)
```

Comment:
Why do this instead of rendering the template and then clearing the session? This is more complicated and now there is data in the session after the user has completed the journey.
Also, why use 'stored_' attributes when the originals have been removed? They could just be re-added.